packages/flutter/lib/src/gestures/events.dart

```diff
@@ -8,51 +8,76 @@ import 'package:flutter/foundation.dart';
 
 export 'dart:ui' show Offset, PointerDeviceKind;
 
-/// The bit of [PointerEvent.buttons] that corresponds to the "primary
-/// action" on any device.
+/// The bit of [PointerEvent.buttons] that corresponds to a cross-device
+/// behavior of "primary operation".
 ///
-/// More specifially,
+/// More specifially, it includes:
 ///
-///  * For touch screen, it's when the pointer contacts the screen.
-///  * For stylus and inverted stylus, it's when the pen contacts the screen.
-///  * For mouse, it's when the primary button is pressed.
+///  * [kTouchContact]: The pointer contacts the touch screen.
```

Comment:
It's too bad about all of these global constants.  I wish they were in a class.  Probably that ship has sailed since it would be a massive breaking change, but I thought I'd just put that out there.